aidial_adapter_bedrock/bedrock.py

```diff
@@ -38,7 +38,9 @@ def _create_invoke_params(self, model: str, body: dict) -> dict:
             "contentType": "application/json",
         }
 
-    async def ainvoke_non_streaming(self, model: str, args: dict) -> dict:
+    async def ainvoke_non_streaming(
+        self, model: str, args: dict
+    ) -> Tuple[dict, Mapping[str, str]]:
```

Comment:
Minor: if you decided to return tuple, maybe let's add some aliases for typing, like

Body = dict
Headers = Mapping[str, str], so it would be easier to remember, what it returns, without actually go and check code?
```python
    async def ainvoke_non_streaming(
        self, model: str, args: dict
    ) -> Tuple[Body, Headers]:
```

Reply:
Also minor: values inside of header could be not only str

But since we're currently only extract one particular header, it doesn't matter

Reply:
Fixed, added Body and Headers type aliases